apps/condo/domains/common/components/Comments/CommentForm.tsx

```diff
@@ -152,7 +159,7 @@ const CommentForm: React.FC<ICommentFormProps> = ({
             <UploadComponent
                 initialFileList={editableCommentFiles}
                 UploadButton={
-                    <Button type='text'>
+                    <Button type='text' eventProperties={{ components: { id: 'comment-file-upload' } }}>
```

Comment:
This will not be supported in UI-kit Button

Reply:
Yup, that's true. But does ui-kit button component support the same visual style as used on the page?